packages/components/src/filters/advanced/style.scss

```diff
@@ -122,19 +124,24 @@
 	@include breakpoint( '<782px' ) {
 		display: block;
 	}
-
-	&.is-english {
-		display: grid;
-		grid-template-columns: 100px 150px auto;
-
-		@include breakpoint( '<782px' ) {
-			display: block;
-		}
-	}
```

Comment:
This `is-english` class is more about sentence structure than an `rtl` issue. The first and second columns have a fixed width and the third occupies the rest of the space.

<img width="612" alt="screen shot 2019-02-13 at 4 13 49 pm" src="https://user-images.githubusercontent.com/1922453/52684385-f1409c80-2faa-11e9-809a-51f3a78fb11a.png">

When the language is not english, we can't rely on a known order, so we abandon the column width definitions.
See https://github.com/woocommerce/wc-admin/pull/457 for a bit more details.